### PR TITLE
fixed typos

### PR DESCRIPTION
**Pull request by guzzit**
Fixed typos in Smart Contracts turorial. 

In the forkless upgrade tutorial, I replaced frame_support::EqualPrivilegeOnly with frame_support::traits::EqualPrivilegeOnly, 
as frame_support::EqualPrivilegeOnly leads to an unresolved import error when trying to build the demo project.

### CODE REVIEW

File: v3/tutorials/08-ink-workshop/c-build-erc20-contract/index.mdx [hunk 1]
```diff
@@ -450,20 +450,20 @@ you have been running the test cases all along.
...
-or error message is returned.
+verify a function performs as we expected by giving it certain inputs and verifying it is returning
+the result we expect. Along the way we also test for edge cases, e.g. how it would handle an empty value,
+or a value that is out of its expected bound, and test for certain error handling mechanisms being executed
+or error messages being returned.
 
-The benefit of having unit tests written are so that once our program get big or during
-future code refactoring, we can run these tests and if we see them pass, we have the confident that
+The benefit of having unit tests written is that once our program gets big or during
+future code refactoring, we can run these tests and if we see them pass, we have the confidence that
```
Comment (jimmychu0807) [inline v3/tutorials/08-ink-workshop/c-build-erc20-contract/index.mdx:461]:
```suggestion
future code refactoring, we can run these tests and if we see them pass, we are confident that
```

**Review by jimmychu0807 — APPROVED** [review]
@guzzit I am good with your changes. Just a minor suggestion on using `confident` over `confidence`, but I leave it as your call whether you want to merge it.

**Comment by guzzit** [comment]
Hi @jimmychu0807, I've commited the suggestion
